cmd/create/admin/cmd.go

```diff
@@ -176,7 +188,25 @@ func run(cmd *cobra.Command, _ []string) {
 	r.Reporter.Infof("It may take several minutes for this access to become active.")
 }
 
-func FindExistingClusterAdminIDP(cluster *cmv1.Cluster, r *rosa.Runtime) (
+// find the htpasswd idp "cluster-admin"
+func FindClusterAdminIDP(cluster *cmv1.Cluster, r *rosa.Runtime) (htpasswdIDP *cmv1.IdentityProvider) {
```

Comment:
@marcolan018 Could you add a unit test for this function and I think the use of named returns is not something we encourage on this project.

Reply:
@robpblake 
added the unit test and made refactoring on the returns for the functions
please help to review again, thanks